types.d.ts

```diff
@@ -3,10 +3,13 @@ import { ContractTransaction } from '@ethersproject/contracts'
 import { JsonRpcProvider } from '@ethersproject/providers'
 
 // --- Simulation configurations ---
+export type GovernorType = 'oz' | 'bravo'
```

Comment:
generally we might prefer enum to string here, but given the small size of the codebase i dont suggest a refactor

Reply:
(implicitly resolved ✅ )